### DISCUSSION TITLE
2nd OpenRadioss Users’ Day, 27th June 2023 in Aachen (Germany)

### DISCUSSION THREAD

**Discussion by MarianBulla** [OpenRadioss/OpenRadioss#1387] [Announcements]
Dear all,
it's my pleasure to announce our 2nd OpenRadioss users meeting 2023 !

It will be on 27th June in Aachen (Germany).

Don't miss this !
Please register here, if you would like to attend: https://web.altair.com/openradioss-users-day-2023

Looking forward to meeting you in Aachen!  :-)

Thanks and best regards,
Marian

**Comment by PaulAltair** [comment]
Update for the event!
Livestream option to attend [#OpenRadioss](https://www.linkedin.com/feed/hashtag/?keywords=openradioss&highlightedUpdateUrns=urn%3Ali%3Aactivity%3A7077893374204104704) Users Day for those across the globe unable to travel. Register here before midnight Monday 26 June CEST https://web.altair.com/openradioss-users-day-2023
